dependencies/vaticle/repositories.bzl

```diff
@@ -21,7 +21,7 @@ def vaticle_dependencies():
     git_repository(
```

Comment:
Are these just to make all the dependencies use Bazel 5 as well?

Reply:
Correct.